consensus/pacemaker/module.go

```diff
@@ -120,18 +120,7 @@ func (m *pacemaker) ShouldHandleMessage(msg *typesCons.HotstuffMessage) (bool, e
 	// 2. If node is synced, node must reject the proposal because proposal is not valid.
 	if msg.Height > currentHeight {
 		m.logger.Info().Msgf("⚠️ [WARN] ⚠️ Node at height %d < message height %d", currentHeight, msg.Height)
-		isSynced, err := m.GetBus().GetConsensusModule().IsSynced()
-		if err != nil {
-			return false, err
-		}
-
-		if !isSynced {
-			err = m.GetBus().GetStateMachineModule().SendEvent(coreTypes.StateMachineEvent_Consensus_IsUnsynced)
-			return false, err
-		}
-
-		return false, nil
-
+		return false, m.GetBus().GetStateMachineModule().SendEvent(coreTypes.StateMachineEvent_Consensus_IsUnsynced)
```

Comment:
Can you assign this to an `err` local var and `return false, err`. 

This is hard to read.